x-pack/plugins/observability/public/components/shared/alert_search_bar/alert_search_bar.test.tsx

```diff
@@ -88,21 +82,20 @@ describe('ObservabilityAlertSearchBar', () => {
   });
 
   it('should filter active alerts', async () => {
-    const mockedSetEsQuery = jest.fn();
+    const mockedOnEsQueryChange = jest.fn();
     const mockedFrom = '2022-11-15T09:38:13.604Z';
     const mockedTo = '2022-11-15T09:53:13.604Z';
-    const { getByTestId } = renderComponent({
-      setEsQuery: mockedSetEsQuery,
-      rangeFrom: mockedFrom,
-      rangeTo: mockedTo,
-    });
 
-    await act(async () => {
-      const activeButton = getByTestId(ACTIVE_BUTTON_DATA_TEST_SUBJ);
-      activeButton.click();
+    act(() => {
```

Comment:
You might not need `act`: https://plainenglish.io/blog/you-probably-dont-need-act-in-your-react-tests-2a0bcd2ad65c

Reply:
Will check 👍🏻